mache/cime_machine_config/config_machines.xml

```diff
@@ -1433,7 +1443,7 @@
         <command name="reset"></command>
         <command name="load">PrgEnv-cray</command>
         <command name="load">craype-accel-amd-gfx90a</command>
-        <command name="load">rocm/5.1.0</command>
+        <command name="load">rocm/5.4.0</command>
```

Comment:
I saw your comment, though I also noticed that you deleted it.

In any case, for our purposes, `frontier` is a machine we support but we treat `frontier-scream-gpu` as a separate machine (even though it is really a different configuration of the same machine) that we don't.  We may be asked to support it in the future but I don't expect that.

Reply:
Thanks! Sorry about the original comment, it was included as part of my "review" of the PR which I accidentally started. It didn't occur to me until now that you might not see my review comments as easily/at all. I just deleted the review and rewrote all my other comments. This was the original comment, for posterity:

I notice we have `frontier_crayclanggpu_mpich.yaml`, is that file referring to the `frontier-scream-gpu` machine? However, when I check `frontier_crayclanggpu_mpich.yaml`, I see that `rocm/5.1.0` has already changed to `rocm/5.4.0`, so we don't need to change anything here. Cue confusion!

Reply:
No, that file refers to the `crayclanggpu` compiler and `mpich` MPI library on `frontier`.

Reply:
> Thanks! Sorry about the original comment, it was included as part of my "review" of the PR which I accidentally started. It didn't occur to me until now that you might not see my review comments as easily/at all.

I can't see comments you make as part of a review until you have posted a review.  Typically, you would not do this for your own PR.

Reply:
I think that must be what happened. I hit the review button by mistake and I guess I never published it